src/Tribe/Tickets.php

```diff
@@ -1580,7 +1658,13 @@ public function process_attendee_meta( $product_id, $meta ) {
 		 */
 		public function get_event_key() {
 			if ( property_exists( $this, 'event_key' ) ) {
-				return $this->event_key;
+				// Seriously?!?!
+				$prop = new ReflectionProperty( $this, 'event_key' );
+				if ( $prop->isStatic() ) {
```

Comment:
anything we can add to this comment to explain it / give context? if not, remove the `// Seriously?!?!` part